test/e2e/mpi_job_test.go

```diff
@@ -164,6 +165,27 @@ var _ = ginkgo.Describe("MPIJob", func() {
 				mpiJob := createJobAndWaitForCompletion(mpiJob)
 				expectConditionToBeTrue(mpiJob, kubeflow.JobSucceeded)
 			})
+
+			ginkgo.It("should not be updated when managed externaly, only created", func() {
+				mpiJob.Spec.RunPolicy.ManagedBy = ptr.To(kubeflow.MultiKueueController)
+				ctx := context.Background()
+				mpiJob = createJob(ctx, mpiJob)
+
+				gomega.Consistently(func() error {
```

Comment:
```suggestion
				gomega.Eventually(func() error {
```

I guess that the `Eventually` should be enough since this verification is just validation testing, right?

Reply:
eventually assumes something will change over time and we wait for the condition, consistently proves us nothing changed - we want the latter

Reply:
That makes sense. Thanks!